bindings/python/src/operator.rs

```diff
@@ -236,6 +236,14 @@ impl Operator {
         ))
     }
 
+    /// Check if this operator can work correctly.
+    pub fn check(&self) -> PyResult<()> {
+        let runtime = pyo3_async_runtimes::tokio::get_runtime();
+        let _guard = runtime.enter();
```

Comment:
this is a temp solution maybe we can find a better solution 
since we have this in build operator isde

Reply:
> It's wired that only this function need workaround. If `check` failed, the `list` should also fail too. I'm guessing all services related to sqlx will fail as well.
> 
> We should identify the root cause and fix it in the core instead. Having such a workaround in the bindings doesn't seem ideal to me.

yes agree and will try to help these days

Reply:
> services related to sqlx will fail as well.
seems yes postgres and mysql are skipped